docs/content/v2024.2/yugabyte-platform/back-up-restore-universes/disaster-recovery/disaster-recovery-tables.md

```diff
@@ -30,6 +34,21 @@ You should perform these actions in a specific order, depending on whether perfo
 | ALTER TABLE<br>ADD CONSTRAINT UNIQUE | Execute on Primary | Execute on Replica | [Reconcile configuration](#reconcile-configuration) |
 | ALTER TABLE<br>DROP CONSTRAINT<br>(unique constraints only) | Execute on Replica | Execute on Primary | [Reconcile configuration](#reconcile-configuration) |
 
+  {{% /tab %}}
+
+  {{% tab header="Semi-automatic schema change mode" lang="semi-automated-schema-changes" %}}
```

Comment:
Nit: Wonder if we should make semi-automatic the default one (though it is ea). So as a path of least resistance, the user will be more persuaded to use the db scoped replicaiton